src/commands/upload_package_using_cli.yml

```
@@ -0,0 +1,22 @@
+description:
```

```suggestion
description: >
```
Add the strip line break to mirror other examples?